uniffi_bindgen/src/bindings/kotlin/gen_kotlin/mod.rs

```diff
@@ -545,29 +545,33 @@ mod filters {
         Ok(as_ct.as_codetype().literal(literal, ci))
     }
 
+    // Get the idiomatic Kotlin rendering of an integer.
+    fn int_literal(t: &Option<Type>, base10: String) -> Result<String, askama::Error> {
+        if let Some(t) = t {
+            match t {
+                Type::Int8 | Type::Int16 | Type::Int32 | Type::Int64 => Ok(base10),
+                Type::UInt8 | Type::UInt16 | Type::UInt32 | Type::UInt64 => Ok(base10 + "u"),
```

Comment:
Is it worth double-checking that `base10` is really base-10 digits?  I don't think it's currently possible, but things might get confusing if it started with something like `0x` .

Reply:
That comes from https://docs.rs/syn/latest/syn/struct.LitInt.html#method.base10_digits, which is why I stuck with the name. I haven't experimented, but the docs for that seem clear.